spec/rails_helper.rb

```diff
@@ -47,13 +48,18 @@
   #
   # The different available types are documented in the features, such as in
   # https://relishapp.com/rspec/rspec-rails/docs
+  config.include Devise::TestHelpers, type: :controller
   config.infer_spec_type_from_file_location!
   config.include FactoryGirl::Syntax::Methods
   config.before(:suite) do
-    DatabaseCleaner.strategy = :transaction
     DatabaseCleaner.clean_with(:truncation)
   end
   config.before(:each) do
+    if Capybara.current_driver == :rack_test
```

Comment:
I thought you installed selenium? I think you have to let Rails know that you want to use the selenium driver. If you're going with poltergeist, just leave it for now